### DISCUSSION TITLE
Is it possible to do absolute positioning?

### DISCUSSION THREAD

**Discussion by WeitingC219** [react-page/react-page#1154] [Q&A]
I'm wondering if it is possible to place plugins with absolute coordinates rather than using its inherited stack/flex layout mechanism. For example, if users want to place videos wherever they click on the react page editor, is there any suggestion to this use case?

**Comment by macrozone** [comment]
@WeitingC219 no, its not possibe. Would be tricky for responsive layouts anyway.
